examples/tutorials/03_ready_to_use_model_tf.ipynb

```diff
@@ -57,7 +57,8 @@
    "source": [
     "[MNIST](http://yann.lecun.com/exdb/mnist/) is a dataset of handwritten digits frequently used as a baseline for machine learning tasks.\n",
     "\n",
-    "Downloading MNIST database might take a few minutes to complete."
+    "Downloading MNIST database might take a few minutes to complete.   \n",
+    "In your experiments you can use `Imagenette` and [CIFAR](https://analysiscenter.github.io/batchflow/api/batchflow.opensets.html#cifar100) datasets as well."
```

Comment:
Why is there no link to Imagenette?

Reply:
Imagenette yet not in batchflow documentation